samples/versioning/Core_7/V2.Subscriber/V2.Subscriber.csproj

```diff
@@ -5,7 +5,8 @@
     <LangVersion>7.3</LangVersion>
   </PropertyGroup>
   <ItemGroup>
-    <ProjectReference Include="..\V1.Messages\V1.Messages.csproj" />
+    <ProjectReference Include="..\V2.Contracts\Versioning.Contracts.csproj" />
+    <ProjectReference Include="..\Versioning.Messages\Versioning.Messages.csproj" />
```

Comment:
Versioning.Messages and V2.Messages should be removed from here right?

Reply:
Yep, thanks, must have been some leftovers left when i removed the project :/

Reply:
```suggestion
```